src/NServiceBus.Core/Hosting/Helpers/AssemblyScanner.cs

```diff
@@ -290,8 +289,7 @@ static string FormatReflectionTypeLoadException(string fileName, ReflectionTypeL
     </assemblyBinding>
 </runtime>";
 
-                sb.AppendFormat(bindingRedirects, nsbAssemblyName.Version.ToString(4));
-                sb.AppendLine();
+                sb.NewLine(string.Format(bindingRedirects, nsbAssemblyName.Version.ToString(4)));
```

Comment:
this would be more efficient if we would use the native Format ability of the StringBuilder but I guess it's a micro-optimization and not worth adding this to `NewLine`.